Makefile

```diff
@@ -33,6 +39,11 @@ IMPT_IMG_NAME=cdi-$(IMPORTER)
 GIT_USER=$(shell git config --get user.email | sed 's/@.*//')
 TAG=$(GIT_USER)-latest
 
+# Preflight Check Defaults
+USE_DOCKER=1
+$(eval DOCKER_OUT=$(shell docker ps))
+export DOCKER_OUT
```

Comment:
don't need to export if set as defined above

Reply:
also, not sure this would work, since it's bash syntax

Reply:
doesn't like the DOCKER_OUT != docker ps